pages/common/gh-api.md

```diff
@@ -0,0 +1,36 @@
+# gh api
+
+> Makes authenticated HTTP requests to the GitHub API and prints the response.
+> More information: <https://cli.github.com/manual/gh_api>.
+
+- Display the subcommand help:
+
+`gh api --help`
+
+- Display the releases for the current repository in JSON format:
+
+`gh api repos/:owner/:repo/releases`
+
+- Create a reaction for a specific issue:
+
+`gh api --header {{Accept:application/vnd.github.squirrel-girl-preview+json}} --raw-field '{{content=+1}}' {{repos/:owner/:repo/issues/123/reactions}}`
```

Comment:
Are you _sure_ that `Accept` header is right haha?

Reply:
seems like it should be `Content-Type`...

Reply:
It's a developers preview since 2016 🤷‍♂️ 🐿️
https://docs.github.com/en/enterprise-server@2.20/rest/reference/reactions#delete-a-reaction
https://developer.github.com/changes/2016-05-12-reactions-api-preview/